.circleci/config.yml

```diff
@@ -2,6 +2,7 @@ version: 2.1
 
 orbs:
   slack: circleci/slack@3.4.2
+  prettier: cfra/prettier@0.1.1
```

Comment:
```
Orb cfra/prettier@0.1.1 not loaded. To use this orb, an organization admin must opt-in to using third party orbs in Organization Security settings.
```

I'm not sure who is an org admin in CircleCI for us so I don't know how to resolve this? I would recommend seeing if we can run this via an npm package or a ruby gem?

Reply:
At least as it applies to JavaScript, there's some prior art in `identity-idp` for this using ESLint, and more specifically [`eslint-plugin-prettier`](https://github.com/prettier/eslint-plugin-prettier) and its `prettier/prettier` rule to enforce formatting.

https://github.com/18F/identity-idp/blob/a846e9ae6cfbb9a3bc20b33236197344615dcda5/.eslintrc#L4
https://github.com/18F/identity-idp/blob/a846e9ae6cfbb9a3bc20b33236197344615dcda5/.eslintrc#L19

I don't have a ton of experience with applying it for other files, but did stumble upon this official Stylelint plugin which has a similar rule:

https://github.com/prettier/stylelint-prettier

Reply:
👍  Got it. Removed prettier from CircleCI. Will add ESLint and it's prettier rules. Also looks like Stylelint will work to set rules for yaml, markdown, html and sass formatting.